x-pack/platform/plugins/shared/fleet/server/routes/package_policy/handlers.ts

```diff
@@ -235,6 +236,16 @@ export const createPackagePolicyHandler: FleetRequestHandler<
   let wasPackageAlreadyInstalled = false;
 
   const spaceId = fleetContext.spaceId;
+
+  if (
+    appContextService.getExperimentalFeatures().disableAgentlessLegacyAPI &&
+    request.body.supports_agentless
+  ) {
+    throw new FleetError(
+      'Creating agentless package policies should be done through the agentless policies API.'
```

Comment:
I think using active voice might be better (even if this is an error message):

```suggestion
      'To create agentless package policies, use the Fleet agentless policies API.'
```

Alternatively, if we stick to passive voice, maybe rephrase to:

```suggestion
      'Agentless package policies should be created through the Fleet agentless policies API.'
```